scripts/model_training/retrain_models_pipeline.py

```
@@ -0,0 +1,201 @@
+    print(f"Status: {'✅ SUCCESS' if success else '❌ FAILED'}")
+    return success
+
+
+def refresh_feature_marts() -> bool:
+    """Refresh feature materialized views."""
+    # Check if feature mart SQL exists and run it
+    feature_sql = ROOT / "sql" / "050_feature_marts.sql"
+    if feature_sql.exists():
+        cmd = ["psql", "-f", str(feature_sql)]
+        return run_command(cmd, "Refresh feature marts")
+    else:
```

Ruff (0.15.10)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)